api/proto/node/node.proto

```diff
@@ -33,6 +33,23 @@ service Retrieval {
 	rpc GetBlobHeader(GetBlobHeaderRequest) returns (GetBlobHeaderReply) {}
 	// Retrieve node info metadata
 	rpc NodeInfo(NodeInfoRequest) returns (NodeInfoReply) {}
+
+	////////////////////////////////////////////////////////////////////////////////////////////////
+	// These RPCs are a work in progress and are not yet fully functional. API may not be stable. //
+	////////////////////////////////////////////////////////////////////////////////////////////////
+
+	// GetChunks retrieves all of the chunks from a specified quorum for a blob held by the node.
+	// Eventually this will replace RetrieveChunks.
+	rpc GetChunks(GetChunksRequest) returns (GetChunksReply) {}
+	// GetChunk retrieves a specific chunk for a blob custodied at the Node.
+	rpc GetChunk(common.ChunkKey) returns (common.ChunkData) {}
+	// GetHeader is similar to RetrieveChunks, this just returns the header of the blob.
+	// Eventually this will replace GetBlobHeader.
+	rpc GetHeader(common.BlobKey) returns (GetBlobHeaderReply) {}
+	// StreamHeaders requests a stream all new headers.
+	rpc StreamHeaders(stream StreamHeadersRequest) returns (stream GetBlobHeaderReply) {}
```

Comment:
We should discuss whether we also want to have a `StreamBlobKeys` endpoint.

Reply:
Repeating result of discussion here. It was decided that having a `StreamBlobKeys` endpoint would probably be useful, but that it was not a necessary component for the initial MVP. We plan on circling back on this question after the MVP launches.

Reply:
Remove it for now if we plan to revisit/decide later

Reply:
@mooselumph's prior comment was about adding an additional RPC. The `StreamHeaders` RPC is one that we are confident we will want to keep.